hearchco_example.yaml

```diff
@@ -3,7 +3,7 @@ server:
   cache:
     type: none
   imageproxy:
-    salt: changemepls
+    secret_key: changemepls
```

Comment:
Please don't use `_` in the field names because it makes env vars for config confusing (since `_` is used to indicate a subfield of an object).